integration/spark/shared/src/main/java/io/openlineage/spark/agent/facets/builder/DatabricksEnvironmentFacetBuilder.java

```diff
@@ -77,6 +77,7 @@ private Map<String, Object> getDatabricksEnvironmentalAttributes(SparkListenerJo
             "user",
             "userId",
             "spark.databricks.clusterUsageTags.clusterName",
+            "spark.databricks.clusterUsageTags.clusterAllTags",
```

Comment:
In future we can think of creating integration test for this feature. For now, this would have been an overkill. 
Please add info in CHANGELOG to let other know about the modified behaviour.

Super happy @Anirudh181001  to see you back here 👍 💯 🥇